examples/99-advanced/customization-trame-toolbar.py

```diff
@@ -0,0 +1,155 @@
+"""
+.. _customize_trame_toolbar_example:
+
+Customize Trame toolbar
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Bring more of the power of trame to the jupyter view.
+"""
+import asyncio
+
+from trame.widgets import vuetify
+import vtk
+
+import pyvista as pv
+from pyvista.trame.ui.vuetify2 import button, select, slider, text_field
+
+###############################################################################
+# Let's first create the menu items we want to add to the trame's toolbar.
+# Here we want a "play" button that will be later connected to a slider
+# through the ``btn_play`` function. The slider itself will represent the
+# "resolution" of the model we will render, a text field where the value of
+# the "resolution" will be displayed.
+# We will also add a dropdown menu to toggle the visibility of the model.
+# The dividers are the same already used to divide and organize the toolbar.
+
+
+def custom_tools():
+    vuetify.VDivider(vertical=True, classes='mx-1')
```

Comment:
I agree that VDivider is usable as is. However, the goal of PyVista is to be able to use the divider without the external import of trame. It would be a simple wrapper, but could you add a function like a button or slider?